com.system76.Popsicle.yaml

```diff
@@ -1,30 +1,35 @@
 app-id: com.system76.Popsicle
 runtime: org.freedesktop.Platform
-runtime-version: '20.08'
+runtime-version: '22.08'
 sdk: org.freedesktop.Sdk
 sdk-extensions:
-- org.freedesktop.Sdk.Extension.rust-stable
+  - org.freedesktop.Sdk.Extension.rust-stable
 build-options:
   append-path: /usr/lib/sdk/rust-stable/bin
   env:
     CARGO_HOME: /run/build/popsicle/cargo
 finish-args:
-- --share=ipc
-- --socket=fallback-x11
-- --socket=wayland
-- --system-talk-name=org.freedesktop.UDisks2
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --system-talk-name=org.freedesktop.UDisks2
 command: popsicle-gtk
 cleanup:
-- /share/icons/hicolor/512x512@2x
+  - /share/icons/hicolor/512x512@2x
 modules:
-- name: popsicle
-  buildsystem: simple
-  sources:
-  - type: archive
-    url: https://github.com/pop-os/popsicle/archive/1.3.0.tar.gz
-    sha256: 8759512b22539a3547b0bf4a0ec17f61f59868c5c10ea7025fcf417790f2869d
-  - generated-sources.json
-  build-commands:
-  - sed -i 's|</launchable>|</launchable>\n<icon type="remote" height="512" width="512">https://raw.githubusercontent.com/pop-os/popsicle/master/gtk/assets/icons/512x512/apps/com.system76.Popsicle.png</icon>|' gtk/assets/com.system76.Popsicle.appdata.xml
-  - make
-  - make install prefix=/app
+  - name: popsicle
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://github.com/pop-os/popsicle
+        #  it's maintained on the master branch
+        branch: master
```

Comment:
Isn't it best to stick with releases here?

I can only think that currently, `1.3.0` wouldn't work with `Freedesktop 22.08`.